app/models/record.rb

```diff
@@ -35,36 +37,37 @@ def convert_rss_to_hash
 
   private
 
-  def parse_single_news_from_xml(xml_item)
-    {
-      author: xml_item.xpath("creator").try(:text),
-      full_version: false,
-      news_type: "news",
-      publication_date: publication_date(xml_item),
-      published_at: publication_date(xml_item),
-      source_url: {
-        url: xml_item.at_xpath("link").try(:text),
-        description: "source url of original article"
-      },
-      data_provider: data_provider,
-      contentBlocks: [
-        {
-          title: xml_item.at_xpath("title").try(:text),
-          body: xml_item.at_xpath("description").try(:text)
-        }
-      ]
-    }
-  end
+    def parse_single_news_from_xml(xml_item)
+      {
+        author: xml_item.xpath("creator").try(:text),
+        title: xml_item.at_xpath("title").try(:text),
```

Comment:
wieso is hier `at_xpath` und bei author is nur `.xpath`?

Reply:
> ich dachte news hätten keinen title, sondern der wird immer aus dem ersten content block genommen. wie kommt es jetzt, dass es doch einen geben soll?
> muss ich in der app dann auch ändern, diesen title zu nehmen, statt aus dem ersten content block? @acousma82

Damit man einen globalen title für einen Artikel hat, muss man ihn hier global setzen und importieren. Er wird für einen unique/doubleten check beim Main App server benötigt

Reply:
Steht auch oben in der Commit Nachricht.

Reply:
Das wird glaube ich nicht (mehr?) benötigt.
Nachrichten-Titel können ja identisch sein, z. B. bei zeitlich wiederkehrenden Ereignissen.